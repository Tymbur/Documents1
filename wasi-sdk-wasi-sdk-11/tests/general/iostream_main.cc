#include <iostream>

int main() {
    std::cout << "hello from C++ main with cout!" << std::endl;
    return 0;
}
