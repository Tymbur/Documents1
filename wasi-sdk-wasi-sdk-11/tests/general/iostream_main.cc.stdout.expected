hello from C++ main with cout!
